src/main/java/uk/gov/hmcts/reform/sscs/model/AppInsights.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@Data
+public class AppInsights {
```

Comment:
This could do with a more descriptive name

Reply:
Point taken and addressed